chapters/publication.tex

```diff
@@ -99,91 +174,259 @@ \section{Collaborating on academic writing}
+Most critically, the ampersand (\texttt{\&}), percent (\texttt{\%}), and underscore (\texttt{\_})
+need to be ``escaped'' (interpreted as text and not code) in order to render.
+This is done by by writing a backslash (\texttt{\textbackslash}) before them,
+such as writing \texttt{40\textbackslash\%} for the percent sign to appear in text.
+Despite this, we believe that with minimal learning and workflow adjustments,
+Overleaf is often the easiest way to allow coauthors to write and edit in \LaTeX\,
+so long as you make sure you are available to troubleshoot minor issues like these.
 
 %------------------------------------------------
 
-\section{Publishing data and code for replication}
-
-Data and code should always be released with any publication.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Publishing_Data}}
-\index{data publication}
-Many journals and funders have strict open data policies,
-and providing these materials to other researchers
-allows them to evaluate the credibility of your work
-as well as to re-use your materials for further research.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Exporting_Analysis}}
-If you have followed the steps in this book carefully,
-you will find that this is a very easy requirement to fulfill.\sidenote{
-\url{https://www.bitss.org/2016/05/23/out-of-the-file-drawer-tips-on-prepping-data-for-publication/}}
-You will already have a publishable (either de-identified or constructed)
-version of your research dataset.
-You will already have your analysis code well-ordered,
-and you won't have any junk lying around for cleanup.
-You will have written your code so that others can read it,
-and you will have documentation for all your work,
-as well as a well-structured directory containing the code and data.
+\section{Preparing a complete replication package}
+
+While we have focused so far on the preparation of written materials for publication,
+it is increasingly important for you to consider how you will publish
+the data and code you used for your research as well.
+More and more major journals are requiring that publications
+provide direct links to both the code and data used to create the results,
+and some even require being able to reproduce the results themselves
+before they will approve a paper for publication.\sidenote{
+  \url{https://www.aeaweb.org/journals/policies/data-code/}}
+If your material has been well-structured throughout the analytical process,
+this will only require a small amount of extra work;
+if not, paring it down to the ``replication package'' may take some time.
+A complete replication package should accomplish several core functions.
+It must provide the exact data and code that is used for a paper,
+all necessary de-identified data for the analysis,
+and all code necessary for the analysis.
+The code should exactly reproduce the raw outputs you have used for the paper,
+and should include no extraneous documentation or PII data you would not share publicly.
```

Comment:
What do we mean by "_no extraneous documentation_"?

Reply:
Don't need that word c24cc8d2b083ec6d45b21ced288532305b253650

Reply:
What do we mean by  "_should include no documentation_"? Shouldn't a replication package include some documentation? It is important that the code runs without a lot of manual instructions in the documentation, but including documentation sounds like a good idea to me.

Reply:
Suggest to reframe this as the replication folder should not include documents that are not directly related to the data itself (e.g. survey firm contracts, enumerator contact information, etc)